frontend/src/scenes/surveys/SurveyAppearance.tsx

```diff
@@ -134,71 +134,83 @@ export function SurveyAppearance({
 }
 
 export function Customization({ appearance, surveyQuestionItem, onAppearanceChange }: CustomizationProps): JSX.Element {
-    const { whitelabelAvailable } = useValues(surveysLogic)
-    const { featureFlags } = useValues(featureFlagLogic)
+    const { whitelabelAvailable, surveysStylingAvailable } = useValues(surveysLogic)
+
     return (
         <div className="flex flex-col">
+            <PayGateMini feature={AvailableFeature.SURVEYS_STYLING}>
```

Comment:
I'll add the feature flags once everything else looks good!

Reply:
pay gate flag added. Basically everything should work as it does currently, and when the flag is enabled, all of this gets gated